controller/workitem_blackbox_test.go

```diff
@@ -559,6 +559,37 @@ func (s *WorkItemSuite) TestUpdateWorkitemWithoutReorder() {
 	assert.Equal(s.T(), wi.Data.Attributes[workitem.SystemOrder], updated.Data.Attributes[workitem.SystemOrder])
 }
 
+// TestReorderDoesNotUpdateAnyOtherWIFields tests that no other workitem fields are updated while reorder
+func (s *WorkItemSuite) TestReorderDoesNotUpdateOtherWIFields() {
+	fxt := tf.NewTestFixture(s.T(), s.DB, tf.CreateWorkItemEnvironment())
+
+	// create workitems
+	payload := minimumRequiredCreateWithTypeAndSpace(fxt.WorkItemTypes[0].ID, fxt.Spaces[0].ID)
+	originalTitle := "Original Title"
+	payload.Data.Attributes[workitem.SystemTitle] = originalTitle
+	payload.Data.Attributes[workitem.SystemState] = workitem.SystemStateClosed
+	_, workitem1 := test.CreateWorkitemsCreated(s.T(), s.svc.Context, s.svc, s.workitemsCtrl, *payload.Data.Relationships.Space.Data.ID, &payload)
+	_, workitem2 := test.CreateWorkitemsCreated(s.T(), s.svc.Context, s.svc, s.workitemsCtrl, *payload.Data.Relationships.Space.Data.ID, &payload)
+
+	titleNotUpdated := "This Title Should Not Be Updated"
+	workitem1.Data.Attributes[workitem.SystemTitle] = titleNotUpdated
+
+	payload2 := minimumRequiredReorderPayload()
+
+	var dataArray []*app.WorkItem // dataArray contains the workitem(s) that have to be reordered
```

Comment:
This can also be removed when `fxt.Workitems[]` is used.

Reply:
@jarifibrahim The test fixture gives me a workitem of workitem.Workitem type but I need app.WorkItem to create reorder payload.